### PR TITLE
Measures routing and measures wrapper

### PR DESCRIPTION
**Pull request by benjaminpaige**
Added dynamic routing for measures
Move types from demo measure into measure directory
Added MeasureWrapper comp

### CODE REVIEW

File: services/ui-src/src/measures/2021/AMM.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+// This is where we will put the measures - the exported comp should be named as the measureId (without the trailing '-XX' )
+// All the measures defined in this directory will have all the props from the MeasureWrapper passed into it
+export const AMM = ({ name, year }: Measure.Props) => {
+  return (
+    <>
+      <p>{name}</p>
+      <p>{year}</p>
+    </>
+  );
+};
```
Comment (13bfrancis) [inline services/ui-src/src/measures/2021/AMM.tsx:12]: I really like this as a convention. Makes things easy to follow

File: services/ui-src/src/components/MeasureWrapper/index.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          <QMR.MeasureButtons
+            handleSave={methods.handleSubmit(handleSave)}
+            handleSubmit={methods.handleSubmit(handleSubmit)}
+            lastSavedText="Saved Moments Ago"
+          />
+        }
+      >
+        <form>
+          <CUI.Box maxW="7xl" as="section">
+            {cloneElement(measure, { name, year })}
```
Comment (13bfrancis) [inline services/ui-src/src/components/MeasureWrapper/index.tsx:71]: I like it!!

**Review by 13bfrancis — APPROVED** [review]
This is good. It will bring in a lot of much needed organization to our frontend for the measures and the handling of them.

File: services/ui-src/src/measures/2021/index.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/*
+When importing a measure it should be a named import and added to the measures object below so that it routes correctly
+the key should be the measure id as a string (with '-XX' included) 
+*/
+
+import { AMM } from "./AMM";
```
Comment (abelzer212) [inline services/ui-src/src/measures/2021/index.tsx:6]: Are we sure that the Adult/Child/Health Home variants are all identical besides name? 
So will we need to specify on import: AMM-AD vs AMM-CH vs AMM-HH?

Comment (benjaminpaige) [reply services/ui-src/src/measures/2021/index.tsx:6]: I am 99% sure, but will confirm the extra 1% also maybe we can just drop the '-' and export from indexs. ill try that instead, that is prob better, good call.

File: services/ui-src/src/components/MeasureWrapper/index.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  this would include:
+  - validating the route params (state, coreset)
+  - querying the measure data
+  - defining the update method
+  - handing top level errors
+  - loading states (maybe?)
+
+  all of the methods defined here can be passed as props to every measure below
+  */
+
```
Comment (jdinh8124) [inline services/ui-src/src/components/MeasureWrapper/index.tsx:31]: Excited to use and consume this mesasureWrapper, it should make our lives easier

**Review by jdinh8124 — DISMISSED** [review]

**Pull request merged**